docs/custom_model_loader.md

```diff
@@ -129,6 +129,10 @@ python3 face_detection.py --batch_size 1 --width 300 --height 300 --input_images
 ```
 
 #### A note on blacklisting the model:
-Even though a model is specified the config file, based on certain conditions, for example license expiry, the model may needs to be disabled. To demonstrate this capability, this sample loader allows the users to specify an optional parameter "enable_file" in "custom_loader_options" in configuration file. The file needs to be present at the same base path as models.
+Even though a model is specified the config file, based on certain conditions, for example license expiry, the model may needs to be disabled. 
```

Comment:
specified the config file -> specified **in** the config file
the model may needs to be disabled -> the model may need to be disabled

I am not sure if instead of:
based on certain conditions, for example license expiry,
under certain conditions, for example license expiry,